responsibleai_text/responsibleai_text/managers/error_analysis_manager.py

```diff
@@ -72,6 +72,8 @@ def __init__(self, model, dataset, is_multilabel, task_type, classes=None):
         if self.task_type in classif_tasks:
             dataset = self.dataset.iloc[:, 0].tolist()
             self.predictions = self.model.predict(dataset)
+            if not isinstance(self.predictions, np.ndarray):
+                self.predictions = np.array(self.predictions)
```

Comment:
this should instead be taken care of in ml_wrappers wrap_model function... similar to tabular case, we shouldn't be adding any extra conversion logic here